src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs

```diff
@@ -74,7 +75,20 @@ internal static class CsvAppFileIO {
             var dec = CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator;
 
             using (var e = await session.BeginEvaluationAsync()) {
-                await e.EvaluateAsync($"write.table({result.Expression}, qmethod='double', col.names=NA, file={file.ToRPath().ToRStringLiteral()}, sep={sep.ToRStringLiteral()}, dec={dec.ToRStringLiteral()})", REvaluationKind.Normal);
+                var fileinfo = await e.EvaluateAsync($"rtvs:::export_to_csv({result.Expression}, sep={sep.ToRStringLiteral()}, dec={dec.ToRStringLiteral()})", REvaluationKind.Normal);
```

Comment:
The second eval is not required. But, in remote scenario if the file size is massive then we might have to consider sending it over the network. For now I will change this to a single eval but this is something we will need to address.